### PR TITLE
[PUBDEV-8280] Fix ModelJsonReader.findInJson for Cases When Path Does Not Exist

### PR DESCRIPTION
**Pull request by mn-mikke**
Before this PR, `ModelJsonReader.readTable` throws a null pointer exception if `findInJson` returns `null` (cannot do `null.isJsonNull`). Similarly, `ModelJsonReader.elementExists` returns `true` in this case.

### CODE REVIEW

**Review by valenad1 — APPROVED** [review]
It looks good to me, thank you!

File: h2o-genmodel/src/main/java/hex/genmodel/attributes/ModelJsonReader.java [hunk 1]
```diff
@@ -483,7 +484,11 @@ protected static JsonElement findInJson(final JsonElement jsonElement, final Str
             } else break;
         }
 
-        return result;
```
Comment (michalkurka) [inline h2o-genmodel/src/main/java/hex/genmodel/attributes/ModelJsonReader.java:486]: looks like https://github.com/h2oai/h2o-3/pull/5775/files#diff-4a16d86a307592634a57c770ec83647ad7867d4d373ea304df64f5defc3074cfR475 became redundant

**Review by michalkurka — APPROVED** [review]
Thanks for the fix @mn-mikke

File: h2o-genmodel/src/main/java/hex/genmodel/attributes/ModelJsonReader.java (hunk 1 above)
Comment (mn-mikke) [reply h2o-genmodel/src/main/java/hex/genmodel/attributes/ModelJsonReader.java:486]: yes, you are right. Thank you!

**Review by wendycwong — APPROVED** [review]

**Review by maurever — APPROVED** [review]

**Pull request merged**